src/System.Net.HttpListener/tests/SimpleHttpTests.cs

```diff
@@ -181,8 +180,7 @@ public void ListenerRestart_BeginGetContext_Success()
             }
         }
 
-        [ConditionalFact]
-        [ActiveIssue(39552)]
+        [Fact]
```

Comment:
This needs to be `[ConditionalFact]`.  That is how the SkipTestException() works. Yes, it looks strange having ConditionalFact with no arguments. But that is how it enables the SkipTestException() to work.  "ConditionalTheory" with no arguments would be needed as well if this was a Theory and used SkipTestException.

Reply:
> Yes, it looks strange having ConditionalFact with no arguments.

Thanks! That's exactly why I changed it. Reverted.